client/starwhale/api/_impl/metric.py

```diff
@@ -40,7 +40,7 @@ def _wrapper(*args: t.Any, **kwargs: t.Any) -> t.Dict[str, t.Any]:
             cr = classification_report(
                 y_true, y_pred, output_dict=True, labels=all_labels
             )
-            _summary_m = ["accuracy", "macro avg", "weighted avg"]
+            _summary_m = ["accuracy", "macro_avg", "weighted_avg"]
```

Comment:
"macro avg" and "weighted avg" are the sklearn lib keys. cr cannot find the new keys.

Reply:
ok，Previously datasotre did not support keys with spaces